src/UWP/ArcGISRuntime.UWP.Viewer/ArcGISRuntime.UWP.Viewer.csproj

```diff
@@ -111,8 +111,19 @@
       <Name>Visual C++ 2015 Runtime for Universal Windows Platform Apps</Name>
     </SDKReference>
   </ItemGroup>
+
+  
   <!-- Screenshots -->
   <ItemGroup>
+    <Content Include="Resources\SyntaxHighlighting\highlight.css">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Resources\SyntaxHighlighting\highlight.pack.js">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Resources\SyntaxHighlighting\screen.css">
```

Comment:
It looks like you can do without `screen.css`. The behavior I'm seeing is that it adds a margin at the top of the page and makes it so you have to scroll all the way down to see the horizontal scroll bar. I think the styles there make more sense in the context of showing a code block on a web page than they do when showing a full-page source browser.